chocolatey/docs
# (#534)(#611) choco-theme 0.2.0

## Description Of Changes

The changes in this PR will implement choco-theme 0.2.0. See the PR in choco-theme for more details on these changes:

https://github.com/chocolatey/choco-theme/pull/284

## Motivation and Context

Choco-theme 0.2.0 defines new styles and classes, and the changes here will ensure everything is implemented as it should.

## Testing

Before testing, please review the [Notable Changes and Known Issues section of the choco-theme 0.2.0 PR](https://github.com/chocolatey/choco-theme/pull/284).

1.  Search for `:memo:` in the source code. You should not find any instances of this. 
2. Do general exploratory testing.
    1. View pages in dark and light modes. 
    1. Since most pages in the docs are the same, and no advanced functionality is present, not every page needs to be tested. Every page should look "correct" and generally the same as before, except for the changes noted on the choco-theme 0.2.0 PR above.

### Operating Systems Testing

n/a

## Change Types Made


## Change Checklist


## Related Issue

* ENGTASKS-2271
* https://github.com/chocolatey/choco-theme/issues/224
* https://github.com/chocolatey/choco-theme/issues/239
* https://github.com/chocolatey/choco-theme/issues/269
* https://github.com/chocolatey/choco-theme/issues/270
* https://github.com/chocolatey/choco-theme/issues/271
* https://github.com/chocolatey/choco-theme/issues/283
* https://github.com/chocolatey/docs/issues/534
* https://github.com/chocolatey/blog/issues/177
* https://github.com/chocolatey/boxstarter.org/issues/22
* https://github.com/chocolatey/home/issues/231
* https://github.com/chocolatey/chocolatey.org/issues/212
* https://github.com/chocolatey/chocolateyfest/issues/19 
* https://gitlab.com/chocolatey/licensing-administration/licensing-services/-/issues/143
* https://gitlab.com/chocolatey/community-infrastructure/community.chocolatey.org/-/issues/1197
* https://github.com/chocolatey/docs/issues/611
* https://github.com/chocolatey/blog/issues/199
* https://github.com/chocolatey/boxstarter.org/issues/33
* https://github.com/chocolatey/home/issues/232
* https://github.com/chocolatey/chocolatey.org/issues/213
* https://github.com/chocolatey/copenhagen_theme/issues/13 
* https://gitlab.com/chocolatey/licensing-administration/licensing-services/-/issues/144
* https://gitlab.com/chocolatey/community-infrastructure/community.chocolatey.org/-/issues/1198
* https://github.com/chocolatey/choco/issues/2930
